dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task210319CreateStorageTable.java

```diff
@@ -38,8 +38,8 @@ public String getPostgresScript() {
                 + "    path        varchar(255) not null,\n"
                 + "    group_name varchar(255) not null,\n"
                 + "    hash       varchar(64) not null,\n"
-                + "    metadata   text not null,\n"
                 + "    mod_date   timestamp without time zone NOT NULL DEFAULT CURRENT_DATE,\n"
+                + "    hash_ref   varchar(64), \n"
```

Comment:
do the sql files need to be modified?

Reply:
This should be a New UT? since a customer could be on latest release and needs this fix, for example our internal servers.

Reply:
> do the sql files need to be modified?

No the SQL files were fine

Reply:
> This should be a New UT? since a customer could be on latest release and needs this fix, for example our internal servers.

ok. Then let's make it a new UT and remove the old one.

Reply:
It means, this UT hasn't made it to any release yet